### PR TITLE
adding the logout page

### PR DESCRIPTION
**Pull request by rajaranjanmr**

### CODE REVIEW

**Comment by mmanjuprasad** [comment]
login UI  pops up wonderfully, its nice for me
Keep a working close button on top of both popups


Rest looks good and can be merged

File: logout/logout.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    <link rel="stylesheet" href="../signup/signup.css">
+    <link rel="stylesheet" href="logout.css">
+    <link rel="stylesheet" href="../cart/cart.css">
+<style>
+    
+</style>
+
+</head>
+<body>
+<div class="container">
```
Comment (AshutoshM29) [inline logout/logout.html]: You can set some z-index value to the nav section
PR review

**Review by AshutoshM29 — APPROVED** [review]
Other than the thing that I have mentioned, the rest looks good to merge!!

**Review by mmanjuprasad — APPROVED** [review]
Can be merged

File: logout/logout.html [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    <div class="top-section">
+        <div class="container-logo">
+            <!--top flex-->
+            <div class="logo"><img src="../images/logo.jpeg"> </div>
```
Comment (sri-rishi) [inline logout/logout.html]: Suggestion: Use alt attribute with < img > tag, It will help in accessibility

File: logout/logout.html [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    </div>
+    <div class="header-cart">
+        <h1>************  You are Logged Out ************</h1>
+        </div>
+            
+
+<div class="button-container button-container-logout">
+    <div><button class="login-btn-ecom" >Login</button></div>
+</div>
+    <div id="id01" class="login-container">
```
Comment (Sahil25061999) [inline logout/logout.html]: id naming can be specific

File: logout/logout.html [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+            </div>
+        
+    </div>
+    <div class="header-cart">
+        <h1>************  You are Logged Out ************</h1>
+        </div>
+            
+
+<div class="button-container button-container-logout">
+    <div><button class="login-btn-ecom" >Login</button></div>
```
Comment (sri-rishi) [inline logout/logout.html]: Suggestion: Work on indentation of file will make code readable

File: signup/signup.css [hunk 5]
```diff
@@ -29,6 +29,9 @@ position: relative;
 left:1rem;
 bottom:0rem;
```
Comment (sri-rishi) [inline signup/signup.css]: Suggestion: Using 0 will work fine

**Review by Sahil25061999 — APPROVED** [review]
Suggestion:  The login button is at the bottom of the page I dont know if its intentional but i will suggest to center it, rest all looks good.

**Review by sri-rishi — CHANGES_REQUESTED** [review]
Good work, just some loop holes which I mentioned in review will make code better

**Review by sri-rishi — APPROVED** [review]

**Pull request merged**